README.md

```diff
@@ -1,5 +1,4 @@
-# TPoS - <small>[LNbits](https://github.com/lnbits/lnbits) extension</small>
-<small>For more about LNBits extension check [this tutorial](https://github.com/lnbits/lnbits/wiki/LNbits-Extensions)</small>
+# TPoS
```

Comment:
Are this changes ok? your this started in an old tpos repo, and got smuggled here?